flytekit/bin/entrypoint.py

```diff
@@ -5,6 +5,7 @@
 import pathlib
 import random as _random
 import traceback as _traceback
+from typing import List
```

Comment:
I need to think about this more.  Can I do this as a separate PR?  I'll write up an issue... also because fast-register doesn't work right now with dynamic tasks.